src/clients/muuntaja/muuntaja.js

```diff
@@ -197,22 +216,22 @@ window.onRecordSwap = function(e){
   console.log('Swap:ing between source and base');
 
   //swap id:s
-  sourceInput.value = baseInput.value;
-  baseInput.value = sourceInput.value;
+  const originalSourceId = sourceInput.value;
+  const originalBaseId = baseInput.value;
+
+  sourceInput.value = originalBaseId;
+  baseInput.value = originalSourceId;
 
   //trigger input event listener to update required values (ie. url parameters)
   sourceInput.dispatchEvent(new Event('input'));
   baseInput.dispatchEvent(new Event('input'));
 
   //swap records around without any search
   if(transformed){
-    const sourceData = transformed.source;
```

Comment:
Tämä, missä recordit swapataan päittäin on hyvä, ei tarvitse trigata recordien hakua uudelleen Alephilta.

Reply:
Fix in new commit.